src/main/java/io/blamer/bot/text/TokenOf.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 Blamer.io
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.blamer.bot.text;
+
+import org.cactoos.Text;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+/**
+ * Token from update.
+ */
+public final class TokenOf implements Text {
+
+  /**
+   * Origin text.
+   */
+  private final String origin;
+
+  /**
+   * Ctor.
+   *
+   * @param update Update to get token
+   */
+  public TokenOf(final Update update) {
```

Comment:
@l3r8yJ primary ctor is missing

Reply:
@h1alexbel this one is primary

Reply:
@l3r8yJ but you are encapsulating the string, so make a ctor for string and in this ctor, reuse the primary one

Reply:
@h1alexbel what's the point of this?

Reply:
@l3r8yJ more systematic class developement, once I will add new ctor it will be easier to reuse primary